action.yml

```diff
@@ -0,0 +1,18 @@
+name: enarxbot
+description: Github Actions project automation for Enarx
+branding:
+  icon: git-commit
+  color: green
+
+inputs:
+  GITHUB_TOKEN:
+    description: "An appropriately-scoped token to authenticate with Github."
+    required: true
+  EVENT_NAME:
+    description: "The event that triggered this run."
+    required: true
```

Comment:
We probably want a variety of other GH environment variables too. At least:
* `GITHUB_ACTOR`
* `GITHUB_REPOSITORY`
* `GITHUB_SHA`
* `GITHUB_REF`
* `GITHUB_HEAD_REF` (optional)
* `GITHUB_BASE_REF` (optional)
* `GITHUB_API_URL`
* `GITHUB_GRAPHQL_URL`

We also probably want the contents of the file at `GITHUB_EVENT_PATH`.

We won't use these variables yet. But it will be much easier to ignore variables we don't need than to fix every repo that uses these actions later.